src/Database/Traits/SoftDelete.php

```diff
@@ -177,6 +183,21 @@ public function restore()
         return $result;
     }
 
+    /**
+     * Update relation pivot table deleted_at column
+     */
+    protected function updatePivotDeletedAtColumn(string $relationName, array $options, string|null $value)
+    {
+        $relation = $this->{$relationName}();
+
+        // get deletedAtColumn from the relation options, otherwise use default
+        $deletedAtColumn = array_get($options, 'deletedAtColumn', 'deleted_at');
+
+        $query = $relation->newPivotQuery()->update([
+            $deletedAtColumn => $value,
+        ]);
```

Comment:
@mjauvin best to use the [`getDeletedAtColumn()` method](https://github.com/wintercms/storm/blob/develop/src/Database/Traits/SoftDelete.php#L267-L275) here:

```suggestion
        $relation->newPivotQuery()->update([
            $relation->getDeletedAtColumn() => $value,
        ]);
```

Reply:
You can't really use the getDeletedAtColumn method as the relation is NOT using the softDelete trait

Reply:
Ah of course, good point.